docs/howto/configure/pangeo-requester-pays-access.md

```diff
@@ -0,0 +1,103 @@
+# Pangeo Data Access via Requester Pays
+
+For some hubs, such as our Pangeo deployments, the communities they serve require access to data stored in other projects.
+Accessing data normally comes with a charge that the folks _hosting_ the data have to take care of.
+However, there is a method by which those making the request are responsible for the charges instead: [Requester Pays](https://cloud.google.com/storage/docs/requester-pays).
+This document shows the steps required to set this method up.
+
+## Setting up Requester Pays Access on GCP
+
+```{note}
+We may automate these steps in the future.
+```
+
+Make sure you are logged into the `gcloud` CLI and have set the default project to be the one you wish to work with.
+
+1. Create a new Service Account
+
+```bash
+gcloud iam service-accounts create requester-pays-sa \
+  --description="Service Account to allow access to Pangeo data stored in the cloud" \
+  --display-name="Requester Pays Service Account"
+```
+
+where `requester-pays-sa` will be the name of the Service Account.
+
+```{note}
+We create a separate service account for this so as to avoid granting excessive permissions to any single service account.
+We may change this policy in the future.
+```
+
+2. Grant the Service Account roles on the project
+
+We will need to grant the [Service Usage Consumer](https://cloud.google.com/iam/docs/understanding-roles#service-usage-roles) and [Storage Object Viewer](https://cloud.google.com/iam/docs/understanding-roles#cloud-storage-roles) roles on the project to the new service account.
+
+```bash
+gcloud projects add-iam-policy-binding \
+  --role roles/serviceusage.serviceUsageConsumer \
+  --member "serviceAccount:requester-pays-sa@PROJECT_ID.iam.gserviceaccount.com" \
+  PROJECT_ID
+
+gcloud projects add-iam-policy-binding \
+  --role roles/storage.objectViewer \
+  --member "serviceAccount:requester-pays-sa@PROJECT_ID.iam.gserviceaccount.com" \
+  PROJECT_ID
+```
+
+where `PROJECT_ID` is the ID of the Google Cloud project, **not** the display name!
```

Comment:
maybe add moustache brackets so it's clear this is variable to be replaced? (e.g. `{{PROJECT_ID}}`)